### ISSUE TITLE
[Suggestion]: Config to make the portal of the Wither Storm open up sooner.

### ISSUE THREAD

**Issue by ghost** [nonamecrackers2/crackers-wither-storm-mod#1687]
### Can I please enter the portal?!

This suggestion comes from an issue I had when playing the WS mod again. After defeating the Symbiote and getting the Command Block Sword in Phase 7, I waited for at least 50-60 minutes for the portal to open up, but it didn't happen. And I had the Evolution Speed set to .1. I had to use commands to make the portal open. 

Having to wait a long time for the portal to open up is not fun when you're decked out in late game gear (Diamond or Netherite), and you just want to the defeat the WS already. 

Given how the Evolution Speed config does nothing for the portal opening, I am simply asking for a config where we can modify the number of consumed entities needed for the portal to open, and by extension, how long it takes for the the WS to evolve to the next phase.

This would be great for those who want a fast paced experience for the mod and want things to be more immediate, so to speak.

**Comment by Herobrine982950291** [comment]
I dont think this is needed.... the hole spawns quick enough for me! (this is what I think....)

**Comment by BluePineappleGuy** [comment]
there is already a config for this

**Comment by ghost** [comment]
> there is already a config for this

What's the config called?

**Comment by BluePineappleGuy** [comment]
> > there is already a config for this
> 
> What's the config called?

something along the lines of "till hole shows" or smth, its usually set to 6, but can be set down to 1, which makes it much faster

**Comment by ghost** [comment]
> something along the lines of "till hole shows" or smth, its usually set to 6, but can be set down to 1, which makes it much faster

![image](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/148708399/510e54e0-3a00-4016-9fc4-f6c00a17ff15)

This thing? I'll try this config out, and if it does make a difference, I'll close the issue.
